types/currency_test.go

```diff
@@ -222,6 +222,10 @@ func TestCurrencyEncoding(t *testing.T) {
 // TestNegativeCurrencyMulRat checks that negative numbers are rejected when
 // calling MulRat on the currency type.
 func TestNegativeCurrencyMulRat(t *testing.T) {
+	if testing.Short() {
+		t.SkipNow()
+	}
+
```

Comment:
They depend on the debug flag being set. testing.Short doesn't set the debug flag